includes/integration.php

```diff
@@ -346,7 +346,7 @@ public function sync_orders_chunk($specific_order_ids = false){
 
                 try {
                     $order = new WC_Order($order_id);
-                    if(!empty($order_id) && !empty($order) && $this->stringIsPresent(get_post_meta($order_id, '_billing_email', true))){
+                    if(!empty($order_id) && !empty($order) && ($this->stringIsPresent(get_post_meta($order_id, '_billing_email', true)) || $this->stringIsPresent(get_post_meta($order_id, '_billing_phone', true)))){
```

Comment:
You should change that a bit, so a person can read what is going on in the `if` statement:
```suggestion
                    $has_email = $this->stringIsPresent(get_post_meta($order_id, '_billing_email', true));
                    $has_phone = $this->stringIsPresent(get_post_meta($order_id, '_billing_phone', true));
                    
                    if(!empty($order_id) && !empty($order) && ($has_email || $has_phone)){
```

Reply:
Refactored.